### PR TITLE
Vs input

### PR DESCRIPTION
**Pull request by tcasey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,10 @@
   "main": "index.js",
   "scripts": {
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "build": "build-storybook -c .storybook -o ./docs"
+    "build": "build-storybook -c .storybook -o ./docs",
+    "prepare-release":
+      "git checkout master && git pull --rebase origin master && npm run docs && lerna updated",
```
Comment (aFluttr) [inline package.json]: Doing the git stuff here seems weird to me. Could be annoying and cause problems.

File: packages/vs-button/build/index.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
+'use strict';
```
Comment (aFluttr) [inline packages/vs-button/build/index.js:1]: Does index.js need to be included now? I had been ignoring it and it was being built on prepublish

File: packages/vs-button/src/index.js [hunk 3]
```diff
@@ -93,9 +95,10 @@ class VSButton extends Component {
       textTransform: 'uppercase',
       border: 'none',
       padding: `15px ${ this.sizeState() }`,
+      height: '48px',
       borderRadius: '4px',
       fontSize: '16px',
-      fontFamily: 'IntCircularProBook',
+      fontFamily: 'IntCircularPro, IntCircularProBook, Arial',
```
Comment (aFluttr) [inline packages/vs-button/src/index.js]: Do we need both IntCircularPro and IntCircularProBook? You can probably get rid of book

**Review by aFluttr — CHANGES_REQUESTED** [review]

File: packages/vs-button/src/index.js (hunk 3 above)
Comment (tcasey) [reply packages/vs-button/src/index.js]: Book was what got imported for story book & the other is what Lumen uses. Aren't you guys using Book in ignite?

File: packages/vs-button/build/index.js (hunk 2 above)
Comment (tcasey) [reply packages/vs-button/build/index.js:1]: That gets statically generated.

File: package.json (hunk 1 above)
Comment (tcasey) [reply package.json]: Yeah we can take it out. This is how "vx" handles stuff but I see no issue with removing it.

File: packages/vs-button/build/index.js (hunk 2 above)
Comment (aFluttr) [reply packages/vs-button/build/index.js:1]: 👍

File: packages/vs-button/src/index.js (hunk 3 above)
Comment (aFluttr) [reply packages/vs-button/src/index.js]: I don't think it matters to ignite. I think it's more important to have them consistent.

File: package.json (hunk 1 above)
Comment (aFluttr) [reply package.json]: I vote to kill it

**Review by aFluttr — APPROVED** [review]

**Pull request merged**